CGATPipelines/pipeline_annotations.py

```diff
@@ -653,8 +653,12 @@ def buildContigSizes(infile, outfile):
     fasta = IndexedFasta.IndexedFasta(prefix)
     outs = IOTools.openFile(outfile, "w")
 
-    for contig, size in fasta.getContigSizes(with_synonyms=False).iteritems():
-        outs.write("%s\t%i\n" % (contig, size))
+    contig_sizes = fasta.getContigSizes(with_synonyms=False)
+    contigs = contig_sizes.keys()
+    contigs.sort()
+
+    for contig in contigs:
+        outs.write("%s\t%i\n" % (contig, contig_sizes[contig]))
```

Comment:
Could this not just be:

for contig, size in sorted(fasta.getContigSizes(...)):